source/JsMaterialX/JsMaterialXGenShader/JsGenContext.cpp

```diff
@@ -0,0 +1,96 @@
+#include <MaterialXGenShader/GenContext.h>
+#include <MaterialXGenShader/ShaderGenerator.h>
+#include <MaterialXGenShader/DefaultColorManagementSystem.h>
+#include <MaterialXFormat/Util.h>
+
+#include <iostream>
+
+#include <emscripten/bind.h>
+
+namespace ems = emscripten;
+namespace mx = MaterialX;
+
+void initContext(mx::GenContext& context, mx::FileSearchPath searchPath, mx::DocumentPtr stdLib, mx::UnitConverterRegistryPtr unitRegistry) {
+    // Initialize search paths.
+    for (const mx::FilePath& path : searchPath)
+    {
+        context.registerSourceCodeSearchPath(path / "libraries");
+    }
+
+    context.getOptions().targetColorSpaceOverride = "lin_rec709";
```

Comment:
We should expose the options class. Maybe in a follow up PR.

Reply:
👍